test/verify/check-machines-dbus

```diff
@@ -1251,6 +1256,7 @@ class TestMachinesDBus(machineslib.TestMachines):
 
         # Check activate button when pool activation is pending
         m.execute("virsh pool-define-as mypool --type netfs --source-host 127.0.0.10 --source-path /mnt/pool --target /mnt/pool")
+        self.addCleanup(m.execute, "virsh pool-destroy mypool; virsh pool-undefine mypool")
```

Comment:
Do you really prefer to addCleanup for each resource. If it was up to me I would rather delete *All* libvirt resources after each test finishes, since this is the state that we start with anyway.
``` sh
for pool in $(virsh pool-list --all --name); do virsh pool-destroy $pool && virsh pool-undefine $pool); done
```

Same for all other resources.

Reply:
Right, I can do this. Bit sad that you and Martin mentioned this now, when this converts the last few test. So I guess I'll reworks all of those already merged cleanups to the "big hammer".

Reply:
Oh I just saw Martins comments :) sorry I didn't review these PRs earlier.

Reply:
I'd start with bind-mounting a per-test temporary directory over /var/lib/libvirt/ and stopping/starting libvirtd around it. Shouldn't that take care of most stuff, except storage pools? (But they can be  in the temporary directory as well)